src/main/webapp/app/quiz/edit/short-answer-question/edit-short-answer-question.component.ts

```diff
@@ -59,13 +59,19 @@ export class EditShortAnswerQuestionComponent implements OnInit, OnChanges, Afte
     /** For visual mode **/
     textParts: String [][];
 
+    backupQuestion: ShortAnswerQuestion;
+
     constructor(
         private artemisMarkdown: ArtemisMarkdown,
         private shortAnswerQuestionUtil: ShortAnswerQuestionUtil,
         private modalService: NgbModal
     ) {}
 
     ngOnInit(): void {
+        /** Create question backup for resets **/
```

Comment:
please comment why you convert it to JSON and then back to an object. I guess this is to make sure you get a real copy of the object (please mention this in the comment)

Reply:
I updated to comment:

 /** Create question backup for resets. We convert it first to JSON and then back to make sure we get a real copy of the object **/